### DISCUSSION TITLE
Suggestions reqired to deploy Production Grade E-Com Application with Nextjs & Prisma2

### DISCUSSION THREAD

**Discussion by shrmaky** [prisma/prisma#3540] [Q&A] [answered]
Hello team, i need some suggestion related to deploying my Ecommerce which is build on Nextjs(React) & Prisma2. Right now my FE is deployed on Vercel & for testing am using Heroku Managed Posgres. There is obvious lag isn processing queries.

Can anyone suggest the best option on AWS/GCP/Azure, which are easy to maintain & non-expensive as well. And also how? Also i looked at AWS-Amplify but in that scenario, there won't be any requirement from Prisma.

**Comment by ryands17** [comment, marked as answer]
Hey @shrmaky 👋 Replied on Slack!

**Reply by SharadKumar** [reply]
@shrmaky you have to keep DB and App deployed in the same region. Heroku Postgres is usually `us-east` while Vercel by default is `us-west`. On a Pro account, you can ensure Vercel is hosting your app in us-east.

**Reply by shrmaky** [reply]
Thanks @SharadKumar

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
